### PR TITLE
Increase the timeout of waiting for node ready in local-up-cluster.sh

### PR DESCRIPTION
**Pull request by qiutongs**
Signed-off-by: Qiutong Song <songqt01@gmail.com>

#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:
As debugging k8s locally, I test it with containerd runtime. This PR is to increase the timeout of waiting for "kubelet get node". 60 seconds are not enough for containerd due to some reason unknown. On my local machine, it took around 120s and therefore I set it to 150s.

#### Which issue(s) this PR fixes:

Fixes # None

#### Special notes for your reviewer:

This was my original PR: https://github.com/kubernetes/kubernetes/pull/99109

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Comment by dims** [comment]
/retest
/approve

please ping me when the CI jobs are green for LGTM.

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -692,11 +692,11 @@ function wait_node_ready(){
   # check the nodes information after kubelet daemon start
   local nodes_stats="${KUBECTL} --kubeconfig '${CERT_DIR}/admin.kubeconfig' get nodes"
   local node_name=$HOSTNAME_OVERRIDE
-  local system_node_wait_time=60
+  local system_node_wait_time=150
```
Comment (hasheddan) [inline hack/local-up-cluster.sh:695]: @dims should we make this configurable for users?

Comment (dims) [reply hack/local-up-cluster.sh:695]: @hasheddan +1

Comment (qiutongs) [reply hack/local-up-cluster.sh:695]: ok, let me make it configurable.

File: hack/local-up-cluster.sh [hunk 2]
```diff
@@ -129,6 +129,10 @@ KUBE_CONTROLLERS="${KUBE_CONTROLLERS:-"*"}"
 # Audit policy
 AUDIT_POLICY_FILE=${AUDIT_POLICY_FILE:-""}
 
+# The timeout setting When waiting node ready with 'kubectl get nodes'
+NODE_READY_TIMEOUT_SECOND=${NODE_READY_TIMEOUT_SECOND:-150}
```
Comment (dims) [inline hack/local-up-cluster.sh:133]: please set this to `60` like it was before. you can use the `NODE_READY_TIMEOUT_SECOND` to 150 in your environment.

Comment (qiutongs) [reply hack/local-up-cluster.sh:133]: But the point of this PR is to increase the default timeout. I assume other developers will have similar experience when running local cluster of containerd.

Is there a good reason to keep 60s?

**Pull request closed without merging**